raiden/raiden_service.py

```diff
@@ -635,6 +651,214 @@ def settle(self, token_address, partner_address):
         return channel
 
 
+class ConnectionManager(object):
```

Comment:
General comment: Can we please move this class and the automatic channel provisioning stuff outside of this monster of a file? `raiden_service.py` has become a huge super file which we should try to cut down and not augment.